WordPressAuthenticator/Unified Auth/GoogleAuthenticator.swift

```diff
@@ -120,11 +121,36 @@ class GoogleAuthenticator: NSObject {
     ///                  The values are updated during the Google process,
     ///                  and returned to the calling view controller via delegate methods.
     ///   - authType: Indicates the type of authentication (login or signup)
-    func showFrom(viewController: UIViewController, loginFields: LoginFields, for authType: GoogleAuthType = .login) {
+    func showFrom(
+        viewController: UIViewController & ASWebAuthenticationPresentationContextProviding,
```

Comment:
Considering we almost always want to use the `viewController.window` in `ASWebAuthenticationPresentationContextProviding`'s implemenation, what do you think implementing the protocol inside this library rather than making it a requirement for the client app?

Reply:
Could you expand on this? Are you suggesting to do something like making `GoogleAuthenticator` conform to `ASWebAuthenticationPresentationContextProviding`, store the input `viewController` somehow, then forwarding `viewController.window` in the protocol implementation?

Reply:
Yes, something like that. We can have an appropriate existing type, or create a new type, to implement the protocol.

So, an alternative option would be, creating a new type:
```
class WebAuthenticationPresentationContext: ASWebAuthenticationPresentationContextProviding {
    let window: UIWindow
}
```

And then, in the `requestAuthorization()` below, using `WebAuthenticationPresentationContext(viewController.window)` as value to the `contextProvidier` argument.

Reply:
Thanks Tony!

Addressed in d5ed7ebd